nixos/modules/services/web-apps/komga.nix

```diff
@@ -0,0 +1,100 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.komga;
+
+in {
+  options = {
+    services.komga = {
+      enable = mkEnableOption "Komga is a free and open source comics/mangas media server";
```

Comment:
Note that this will be added to the string `Whether to enable`, so `Whether to enable Komga is a free and open source comics/mangas media server` doesn't really make sense.